src/components/search/SearchResultsContainer.jsx

```diff
@@ -21,7 +21,7 @@ const SearchResultsContainer = ({ currentPage, onPageChange, results }) => {
   const formattedResults =
     results.items[0] &&
     results.items.map(item => {
-      const htmlUrl = item.html_url.split('/');
+      const { htmlUrl, comments } = [item.html_url.split('/'), item.comments];
```

Comment:
Would it be possible to do this on a separate line than the htmlUrl - 
`const { comments } = item;`

Reply:
I used to do it this way, 
![ss03](https://user-images.githubusercontent.com/60892464/96352155-28742480-108f-11eb-9675-66f02f1c633b.jpg)

but when I run the formating command, it said 
![ss04](https://user-images.githubusercontent.com/60892464/96352176-4772b680-108f-11eb-83d5-716c4b7ddc69.jpg)

Reply:
Oh gotcha. Please skip the lint:fix command for now